### PR TITLE
docs: add why account kit doc

### PR DESCRIPTION
**Pull request by SahilAujla**
- TODO: Add links to signers like Lit Protocol, Privy etc. once they are added in docs.
- TODO: Add [ HERO GIF/VIDEO ]

https://github.com/OMGWINNING/aa-sdk-staging/assets/83442423/8dd92d97-6c97-43c6-bdad-8eb6e54f9a3b

### CODE REVIEW

File: site/why-account-kit.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+  - - meta
+    - property: og:description
+      content: Learn why we decided to build Account Kit and how it can help you provide seamless UX to your users
+---
+
+# Why Account Kit
+
+## The Problem
+
+It’s way too hard for new users to start using your app. Today, users have to jump through hoops before they submit their first transaction:
```
Comment (rthomare) [inline site/why-account-kit.md:19]: It needs to be simpler for users to use existing web3 DApps today. Users have too many steps before they can even submit their first transaction!

File: site/why-account-kit.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+## The Solution: Smart Accounts
+
+Account Kit provides all the tools you need to onboard the next 1B users with a simple, familiar user experience.
+
+[ HERO GIF/VIDEO ]
+
+With Account Kit, you can create a **smart account** for every user. Smart accounts are smart contract wallets that leverage account abstraction to radically simplify every step of the onboarding experience. Now, a new user will:
+
+1. Create an account natively in your app with no third-party downloads
```
Comment (rthomare) [inline site/why-account-kit.md:36]: Create their web3 account directly in your app without third-party downloads

File: site/why-account-kit.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+Account Kit makes it possible to build a web3 app that feels like web2: simple and familiar for mainstream users.
+
+It’s enabled by [ERC-4337](https://eips.ethereum.org/EIPS/eip-4337), a new account abstraction standard developed by Vitalik and the Ethereum Foundation. This is the beginning of a major [transition](https://vitalik.ca/general/2023/06/09/three_transitions.html) from traditional EOA wallets to smart accounts.
+
+Account Kit was designed to make it easy for every developer to leverage account abstraction. It includes everything you need to ship smart accounts in minutes!
+
+## Your app, your UX
+
+With Account Kit, you can design every step of the user experience from signup to checkout, natively in your app. Instead of sending a signature request to a third-party wallet, you can display the confirmation natively in your app or even submit the transaction invisibly in the background. With this level of control, wallets fade into the background. You can curate exactly what information to show the user to help them complete their journey in the app.
```
Comment (rthomare) [inline site/why-account-kit.md:49]: This feels too wordy can we make this shorter?

_Account Kit streamlines the user experience in your app from signup to checkout. It replaces third-party wallet interactions with native confirmations and background transactions, reducing traditional wallet visibility. You control the information displayed and navigate users within your application._

File: site/why-account-kit.md [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+
+Account Kit integrates all the leading wallet Signers with bespoke integration guides for [Magic.link](/smart-accounts/signers/magic-link), [web3auth](/smart-accounts/signers/magic-link), [Turnkey](/smart-accounts/signers/turnkey), Privy, Lit Protocol, Fireblocks, Portal, and [Capsule](/smart-accounts/signers/capsule). Account Kit even supports self-custodial wallets like Metamask or Ledger. Users can even change their signer later via Account Kit’s [ownership transfer](/smart-accounts/transferring-ownership) functionality.
+
+Learn [how to choose the right signer for your use case in this doc](/smart-accounts/signers/overview).
+
+## Sponsor gas
+
+Account Kit removes the greatest barrier to entry of all: gas fees.
+
+Countless crypto-curious newcomers have browsed NFTs, but many of them gave up before actually buying one. Even if the NFT is free, gas fees are a major deterrent because buying crypto for the first time is intimidating. You need to put down a credit card, submit personal information like an ID, and then transfer tokens from an exchange. Millions of users have dropped off and abandoned web3 at this step.
```
Comment (rthomare) [inline site/why-account-kit.md:70]: General thoughts here is this feels a bit wordy can we shorten the content in this doc?

The daunting process of the initial crypto purchase. It involves credit card use, personal data submission, and token transfers. This discourages incoming users, leading to a significant drop-off among web3 newcomers.

File: site/why-account-kit.md [hunk 5]
```diff
@@ -0,0 +1,104 @@
+---
```
Comment (rthomare) [inline site/why-account-kit.md:1]: This content generally makes sense, but can we take another pass with a knife and shorten the verbage? It could just be me but I think there is a chance to lower the raw number of words.

File: site/why-account-kit.md (hunk 1 above)
Comment (willhennessy) [reply site/why-account-kit.md:19]: hm i prefer the original because it's important to describe the problem rather than the solution. "it needs to be simpler" is more speaking about the solution

i would tweak to "start using web3 apps".  otherwise ship it!

File: site/why-account-kit.md (hunk 2 above)
Comment (willhennessy) [reply site/why-account-kit.md:36]: thanks rohan!  final:

Create a smart account directly in your app without third-party downloads

File: site/why-account-kit.md (hunk 3 above)
Comment (willhennessy) [reply site/why-account-kit.md:49]: good push to simplify. side note: I agree the above is wordy, but don't want to lose the specific and concrete language. "streamlined" is a bit hand wavy

Let's revise to:

With Account Kit, you can design every step of the user experience from signup to checkout natively in your app. Replace third-party wallet interactions with native confirmations and background transactions. Curate the right information to display and abstract away the details. Use Account Kit to streamline the experience and improve engagement with your app.

File: site/why-account-kit.md (hunk 4 above)
Comment (willhennessy) [reply site/why-account-kit.md:70]: thanks good push again. 

Many newcomers give up on web3 before submitting their first transaction. It's daunting to buy crypto for the first time, especially before trying the app! Gas fees -- even cheap ones on L2 -- discourage newcomers from trying your app.

File: site/why-account-kit.md (hunk 5 above)
Comment (willhennessy) [reply site/why-account-kit.md:1]: agreed and accepted most of the above. i've spent too much time on this doc already, so let's merge it and try to shorten later

File: site/why-account-kit.md (hunk 1 above)
Comment (SahilAujla) [reply site/why-account-kit.md:19]: Changed to "start using web3 apps"

File: site/why-account-kit.md (hunk 3 above)
Comment (SahilAujla) [reply site/why-account-kit.md:49]: Changed!

File: site/why-account-kit.md (hunk 4 above)
Comment (SahilAujla) [reply site/why-account-kit.md:70]: Changed!

**Comment by SahilAujla** [comment]
Incorporated all the feedback, and we're good to merge!

**Review by moldy530 — APPROVED** [review]
approving and merging so this is ready for dogfooding. asana task created to track putting the hero video here: https://app.asana.com/0/0/1205684445807209/f
